README.md

```diff
@@ -73,9 +73,9 @@ To build `workerd`, you need:
   * If you use [Bazelisk](https://github.com/bazelbuild/bazelisk) (recommended), it will automatically download and use the right version of Bazel for building workerd.
 * On Linux:
   * We use the clang/LLVM toolchain to build workerd and support version 15 and higher. Earlier versions of clang may still work, but are not officially supported.
-  * Clang 15+ (e.g. package `clang-15` on Debian Bookworm).
+  * Clang 15+ (e.g. package `clang-15` on Debian Bookworm). If clang is installed as `clang-<version>` please create a symlink for just `clang` too or use `--action_env=CC=clang-<version>` to specify the compiler name.
   * libc++ 15+ (e.g. packages `libc++-15-dev` and `libc++abi-15-dev`)
-  * LLD 15+ (e.g. package `lld-15`).
+  * LLD 15+ (e.g. package `lld-15`). The build may still succeed if a recent version of GNU gold or the system linker is installed, but lld is highly recommended for link performance.
```

Comment:
I wonder... Is bazel smart enough to find `lld-15` if it is installed, or would it need a similar symlink as `clang` does?

Reply:
Good question – bazel itself is not smart enough but when configuring the linker it invokes the output of a `clang -v` link action, which allows it to capture the linker path used by clang, allowing it to indirectly get the right linker version. This is hidden somewhere deep in the bazel code base, would provide a link but I'm on mobile ATM.

Reply:
Ah, nice, sounds like it wouldn't be an issue, then.